model/map-hot-rod/src/main/resources/config/cacheConfig.xml

```diff
@@ -6,5 +6,10 @@
         <distributed-cache name="clients" mode="SYNC">
             <encoding media-type="application/x-protostream"/>
         </distributed-cache>
+        <distributed-cache name="groups" mode="SYNC">
```

Comment:
Can you please add the new cache to `testsuite/model/src/main/resources/hotrod/infinispan.xml` and `model/map-hot-rod/src/main/resources/config/infinispan.xml`?